code-challenges/401challenges/src/main/java/challenges/linkedlist/LinkedList.java

```diff
@@ -0,0 +1,72 @@
+package challenges.linkedlist;
+
+import java.util.ArrayList;
+
+public class LinkedList {
+   public  Node head;
+   public int length;
+
+    public void LinkedList(){
```

Comment:
The `void` is not necessary and is technically not true. This is the constructor which is returning something, an instance of the LinkedList class